src/shared/services/search/search-facade.service.ts

```diff
@@ -0,0 +1,12 @@
+import { Injectable } from '@angular/core';
+import { Store } from '@ngrx/store';
+import { SearchActions } from '../../store';
+
+@Injectable()
+export class SearchFacadeService {
+  constructor(private store: Store) {}
+
+  searchTodo(searchTerm: string): void {
```

Comment:
it's ok for this project, but if you want to make a reusable service, it shouldn't depend on todos in naming